hedera-node/hedera-app/src/main/java/com/hedera/node/app/info/NumbersModule.java

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright (C) 2023 Hedera Hashgraph, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.hedera.node.app.info;
+
+import com.hedera.node.app.spi.numbers.HederaFileNumbers;
+import dagger.Binds;
+import dagger.Module;
+
+@Module
+public interface NumbersModule {
```

Comment:
For clarity, could we name this `FileNumbersModule`, so it's clear we're supporting file numbers?

Reply:
done thanks